### ISSUE TITLE
[Edit] - "Function factories"-Chapter

### ISSUE THREAD

**Issue by Tazinho** [Tazinho/Advanced-R-Solutions#136]
@Tazinho 
- [x] Sync subsection names
- [x] Ensure that exercise numbers go 1,2,3,.. in each subsection.
- [x] Ensure linebreaks between Q & As
- [x] Sync the text and the order of all exercises in the chapter between Adv R and Adv R Solutions.

Read through all exercises. Edit them if possible and mark their correct status (green = "think this solution is correct", orange = "this solution is definitely not correct or sth. is missing". If sth. is missing, try to includede a todo). Also document open points within the following checklist:

### [Factory funfamentals](https://advanced-r-solutions.rbind.io/function-factories.html#factory-fundamentals)
- [x] 2. Base R contains two function factories, approxfun() and ecdf(). Read their documentation and ...
  - Needs an introduction phrase.
  - Needs to go a bit more into detail (and name things more clearly).
- [x] 3. Create a function pick() that takes an index, i, as an argument and returns a function with an ...
  - Needs an introduction phrase.
- [x] 4. Create a function that creates functions that compute the ith central moment of a numeric vector
  - Not sure if we should leave the link within the answer. @henningsway What do you think?
### [Graphical factories](https://advanced-r-solutions.rbind.io/function-factories.html#graphical-factories)
- [ ] 1. Compare and contrast ggplot2::label_bquote() with scales::number_format()....
  - Needs to be solved
### [Statistical factories](https://advanced-r-solutions.rbind.io/function-factories.html#statistical-factories)
- [x] 1. In boot_model(), why don’t I need to force the evaluation of df or model?...
  - Needs to include the context of the exercise. Otherwise it is hard to follow.
- [ ] 2. Why might you formulate the Box-Cox transformation like this?...
  - Needs to be solved/finished (I just added my thoughts as a "started" solution)
- [x] 4. How much time does ll_poisson2() save compared to ll_poisson1()? Use...
  - Could possibly be prettified a bit.
### [Function factories + functionals](https://advanced-r-solutions.rbind.io/function-factories.html#function-factories-functionals)
- [ ] 2. Compare and contrast the effects of env_bind() vs. attach() for the following code....
  - Needs to be solved.
- [x] All open points/exercises should now be reported explicitly under each bullet point above (maybe henningsway can solve one or the other, has a good idea for a new approach or has a more specific question to raise in an issue to Hadley. 

- [x] Assign Henningsway

@henningsway 
- [x] Read & edit all exercises.
- [x] Report any new points while updating the list above. 
- [x] Run a final spellcheck

@Tazinho and @henningsway 
- [ ] Try to resolve any of the reported issues above.

**Comment by Tazinho** [comment]
@henningsway I am mostly done with this. I would suggest that you do the remaining editing (only what is marked as checkbox) and try to solve any of the 3 open exercises . If you can not, it is also fine. Afterwards pls close here and assign Hadley for the review issue. Hope that is okay?

**Comment by henningsway** [comment]
I edited the solved exercises. We still need to tackle a couple of unsolved ones.

**Comment by henningsway** [comment]
Alright 3 exercises still need to be solved. Let's leave these open for now and tackle them later.

We can track them via our progress chart. I will close here for now.

**Comment by Tazinho** [comment]
@henningsway solved the last one. Maybe you can edit it and afterwards set up the review issue for this chapter?
